app/components/scheduled_post_indicator/scheduled_post_indicator.tsx

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+import {Text, View} from 'react-native';
+
+import CompassIcon from '@components/compass_icon';
+import FormattedTime from '@components/formatted_time';
+import {useTheme} from '@context/theme';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+import {getUserTimezone} from '@utils/user';
+
+import type UserModel from '@typings/database/models/servers/user';
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
+    return {
+        container: {
+            backgroundColor: changeOpacity(theme.centerChannelColor, 0.08),
+            paddingVertical: 12,
+            paddingHorizontal: 16,
+            color: changeOpacity(theme.centerChannelColor, 0.72),
+            display: 'flex',
+            flexDirection: 'row',
+            alignItems: 'center',
+            gap: 12,
+        },
+        text: {
+            color: changeOpacity(theme.centerChannelColor, 0.75),
+            fontSize: 14,
+        },
+        link: {
+            color: theme.linkColor,
+            fontSize: 14,
+        },
+    };
+});
+
+type Props = {
+    currentUser?: UserModel;
+    isMilitaryTime: boolean;
+}
+
+export function ScheduledPostIndicator({currentUser, isMilitaryTime}: Props) {
+    const theme = useTheme();
+    const styles = getStyleSheet(theme);
+
+    const [scheduledPostCount] = React.useState(125);
```

Comment:
This 125 is hardcoded just for demo. It will later be replaced with actual count.

Reply:
In this PR, or in a future PR?

Reply:
In the next week or two. It will be in there before we merge feature branch into `main`.

Reply:
Please put a TODO comment that this value needs to be replaced.  Just so that it won't get forgotten.

Reply:
scheduledPostCount is now. props, so this is removed now.